### ISSUE TITLE
Enough data to finetune?

### ISSUE THREAD

**Issue by danmodeller57** [shiyu-coder/Kronos#171]
Hi,

Do I have enough data to finetune the kronos model, and to forecast with the finetuned model? I only have this timeseries!

<img width="693" height="449" alt="Image" src="https://github.com/user-attachments/assets/a8f900e2-4320-4b80-b76b-49ed7ca7dcd7" />

**Comment by shiyu-coder** [comment]
Perhaps it's possible, you can try fine-tuning with a smaller learning rate (such as 1e-5 or even lower) and observe the performance.
